applications/vanilla/modules/class.usercommentsmodule.php

```diff
@@ -0,0 +1,79 @@
+<?php
+/**
+ * User comments module.
+ *
+ * @copyright 2009-2017 Vanilla Forums Inc.
+ * @license http://www.opensource.org/licenses/gpl-2.0.php GNU GPL v2
+ * @package Vanilla
+ * @since 2.3
+ */
+
+/**
+ * Renders recently posted comments from a specific user
+ */
+class UserCommentsModule extends Gdn_Module {
+
+    /** @var int Display limit. */
+    public $limit = 10;
+
+    /** @var  int The ID of the user for whom you are showing the comments. */
+    public $userID = null;
+
+    /**
+     *
+     *
+     * @throws Exception
+     */
```

Comment:
Need some kind of description for this doc block.  Also needs a `@param` tag for `$sender`.